src/ProjectSystem.sln

```diff
@@ -325,4 +325,7 @@ Global
 		{CC1FF91F-C0A9-4729-9863-45A688483F84} = {7AC93D71-E528-4BDA-AEEF-47876EC5F5A0}
 		{AE328174-EBB5-49CF-B642-DBF6CEB2EA4B} = {6DE4B49E-AC38-4E35-82CD-D52731503111}
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
```

Comment:
VS has now started persisting a "SolutionGuid" in 15.3 - I've not tested if this is backwards compatible but it would be a bug if it wasn't.